sci2zero/TeslaRIS-frontend
# Feature/deduplication

## Description

Added deduplication page as well as comparators for all remaining entities. Refined some forms, refined merging process, added deletion during comparison and added some QoL improvements.

## How should this be tested?

This should be tested in conjunction with [corresponding backend PR](https://github.com/sci2zero/TeslaRIS-backend/pull/105). When logged in as admin, navigate to deduplication page and start the process. Once it is completed you should receive a notification. Merging process should be done from deduplication page intuitively. Merging from browsing pages is also possibl.

## Checklist
